ingestion/src/metadata/ingestion/ometa/mixins/lineage_mixin.py

```diff
@@ -52,11 +52,12 @@ class OMetaLineageMixin(Generic[T]):
     def _merge_column_lineage(
         self, original: List[Dict[str, Any]], updated: List[Dict[str, Any]]
     ):
-        temp_result = []
+        flat_original_result = set()
```

Comment:
Are we using `set` to get rid of duplicate entries ?

Reply:
yes